pkgs/by-name/ga/garnet/package.nix

```diff
@@ -20,12 +20,7 @@ buildDotnetModule rec {
   projectFile = "main/GarnetServer/GarnetServer.csproj";
   nugetDeps = ./deps.nix;
 
-  dotnet-sdk =
-    with dotnetCorePackages;
-    combinePackages [
-      sdk_6_0
-      sdk_8_0
-    ];
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
```

Comment:
Same thing with this package too.

Also the same change is being done in https://github.com/NixOS/nixpkgs/pull/361658